src/main/java/org/alliancegenome/curation_api/services/helpers/annotations/GeneExpressionAnnotationUniqueIdHelper.java

```diff
@@ -8,12 +8,12 @@
 public class GeneExpressionAnnotationUniqueIdHelper {
 
 	//	UniqueID =
-	//	assayId | evidenceCrosseferenceId | geneId | stageTermId | stageName | whereExpressedStatement | anatomicalStructureTermId | cellularComponentTermId
-	public String generateUniqueId(GeneExpressionFmsDTO geneExpressionFmsDTO) {
+	//	assayId | evidenceCurie | geneId | stageTermId | stageName | whereExpressedStatement | anatomicalStructureTermId | cellularComponentTermId
+	public String generateUniqueId(GeneExpressionFmsDTO geneExpressionFmsDTO, String curie) {
```

Comment:
I would rename this parameter as `referenceCurie` rather than just `curie` so that devs can see exactly what this refers to without having to trace back to the method call

Reply:
I guess the comment does this already, but the parameter name makes it clearer when it is used later in the method

Reply:
Renamed